WordPress/Classes/Models/Blog.h

```diff
@@ -60,17 +60,24 @@ typedef NS_ENUM(NSUInteger, BlogFeature) {
 @property (nonatomic, strong, readwrite) WPAccount      *jetpackAccount;
 @property (nonatomic, assign, readwrite) BOOL           videoPressEnabled;
 @property (nonatomic, assign, readwrite) BOOL           isMultiAuthor;
-@property (nonatomic, assign, readwrite) BOOL           isJetpack;
+@property (nonatomic, assign, readwrite) BOOL           isHostedAtWPcom;
 @property (nonatomic, strong, readwrite) NSString       *icon;
+/**
+ Stores the username for self hosted sites
+ 
+ @warn For WordPress.com or Jetpack Managed sites this will be nil. Use usernameForSite instead
+ */
+@property (nonatomic, strong,  readwrite) NSString       *username;
```

Comment:
There is some extra whitespace before `readwrite`. This file's properties look really scrambled since there are a lot of extra whitespace for alignment. We should make a pass at this at some time, so we don't need to worry about aligning properties anymore.